tajo-core/src/test/java/org/apache/tajo/engine/query/TestWindowQuery.java

```diff
@@ -462,4 +462,148 @@ public final void testMultipleWindow() throws Exception {
       executeString("DROP TABLE multiwindow PURGE");
     }
   }
+
+  @Test
+  public final void testWindowFrame1() throws Exception {
```

Comment:
You're right, @jihoonson.
I'll separate codes, queries, and results.

By the way, in other part of Tajo code, 
there exists test codes like what I did (for example, testJoinQuery, testJoinBroadcast, ...).
I think we need overall refactoring of test code to make them follow our test writing convention.